supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesJobPaths.scala

```diff
@@ -31,6 +31,7 @@ final case class JesJobPaths(override val workflowPaths: JesWorkflowPaths, jobKe
 
   val jesMonitoringScriptFilename: String = s"${JesJobPaths.JesMonitoringKey}.sh"
 
+  // This comment appears to be vestigial.
```

Comment:
And by 'this' I mean 'that', the comment below 'this'.

Reply:
I'd agree and since I can't readily find its home I'd assume it's orphaned.  IMO it can go.